comma/model.py

```diff
@@ -33,30 +38,103 @@ def setup(self, size: int, dir_params: str,
         self.dir_params = dir_params
         Hypothesis.validate_param_file(dir_params)
         if use_ipf:
-            self.agents = Individual.populate_ipf(size, dir_params)
+            self.agents = Individual.populate_ipf(size, dir_params, rng=self.rng)
         else:
-            self.agents = Individual.populate(size, dir_params)
+            self.agents = Individual.populate(size, dir_params, rng=self.rng)
+
+    def update_covid_counter(self):
+        """
+        Update the days_since_positive counter
+        for agents with covid_status of 1.
+        Returns:
+            None. This function updates the agent's covid counter in place.
+        """
+        for agent in self.agents:
+            if agent.covid_status == 1:
+                agent.days_since_positive += 1
```

Comment:
Assuming that `agent.covid_status` can only be `0` or `1`, this should be equivalent to:

```suggestion 
            agent.days_since_positive += agent.covid_status
```
Anyway, it might be clearer if `agent.covid_status` was a boolean instead of an integer (implicitly the same thing as `0` or `1`).
If there are more possible `covid_status` codes, it might be good to name them in an [enum](https://docs.python.org/3/library/enum.html).